Tests/S4TFSpectralNormalization/S4TFSpectralNormalizationTests.swift

```diff
@@ -1,3 +1,4 @@
+// TODO (crcpar): Write tests.
```

Comment:
- Check SNConv2D behaves differently according to `learningPhase`
    - u & v are updated